src/EditorFeatures/TestUtilities/LanguageServer/AbstractLanguageServerProtocolTests.cs

```diff
@@ -431,6 +478,41 @@ internal TestLspServer(TestWorkspace testWorkspace)
                     _executionQueue, methodName, request, clientCapabilities, clientName, cancellationToken);
             }
 
+            public async Task OpenDocumentAsync(Uri documentUri)
```

Comment:
there were multiple implementations of doc sync in tests, unifying them all here